File: Python/stage1/S1_SW_02.py
# S1_SW_02 — Command router
def run_command(cmd):
   match cmd:
       case "start":
           return "System starting"
       case "stop":
           return "System stopping"
       case "status":
           return "System status: OK"
       case _:
           return "Unknown command"
# Tests
commands = ["start", "stop", "status", "restart"]
for c in commands:
   print(c, "=>", run_command(c))

File: js/stage1/S1_SW_02.js
// S1_SW_02 — Command router
function runCommand(cmd) {
  switch (cmd) {
    case "start":
      return "System starting";
    case "stop":
      return "System stopping";
    case "status":
      return "System status: OK";
    default:
      return "Unknown command";
  }
}
// Tests
["start", "status", "stop", "restart"].forEach((c) =>
  console.log(c, "=>", runCommand(c))
);
